recipes/pyjuliaup/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pyjuliaup" %}
+{% set url_name = "juliaup" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ url_name[0] }}/{{ url_name }}/juliaup-{{ version }}.tar.gz
+  sha256: 69abc0bc94e0a1b5717fce9d38547f44e6d88265f81e1a86a3c043e6ce0578ec
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - juliaup
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/juliaup/
+  summary: An interface to the
```

Comment:
Seems tuncated.

Reply:
I believe [grayskull](https://github.com/conda-incubator/grayskull) missed that because the next word has a link... 

cc @marcelotrevisani for viz

Reply:
Should've been: "A Python interface to [JuliaUp](https://github.com/JuliaLang/juliaup), the Julia version manager." Will fix when review is over.